x-pack/test/stack_functional_integration/apps/monitoring/_monitoring_metricbeat.js

```diff
@@ -4,41 +4,88 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+import expect from '@kbn/expect';
 
 export default ({ getService, getPageObjects }) => {
   describe('monitoring app - stack functional integration - suite', () => {
     const browser = getService('browser');
     const PageObjects = getPageObjects(['security', 'monitoring', 'common']);
-    const log = getService('log');
     const testSubjects = getService('testSubjects');
-    const isSaml = !!process.env.VM.includes('saml') || !!process.env.VM.includes('oidc');
     const clusterOverview = getService('monitoringClusterOverview');
     const find = getService('find');
 
     before(async () => {
-      await browser.setWindowSize(1200, 800);
-      if (process.env.SECURITY === 'YES' && !isSaml) {
-        await PageObjects.security.logout();
-        log.debug('### log in as elastic superuser to enable monitoring');
-        // Tests may be running as a non-superuser like `power` but that user
-        // doesn't have the cluster privs to enable monitoring.
-        // On the SAML config, this will fail, but the test recovers on the next
-        // navigate and logs in as the saml user.
-      }
-      // navigateToApp without a username and password will default to the superuser
+      await browser.setWindowSize(1200, 1200);
       await PageObjects.common.navigateToApp('monitoring', { insertTimestamp: false });
       await clusterOverview.acceptAlertsModal();
     });
-
-    it('should have Monitoring already enabled', async () => {
+    beforeEach(async () => {
+      await PageObjects.common.navigateToApp('monitoring', { insertTimestamp: false });
+      //await clusterOverview.acceptAlertsModal();
```

Comment:
Can this comment also be removed?  I'm not sure what modal that was.  The one to enable monitoring?